src/Database/Database.php

```diff
@@ -114,6 +114,7 @@ public function connect()
 		$pass    = trim($this->configCache->get('database', 'password'));
 		$db      = trim($this->configCache->get('database', 'database'));
 		$charset = trim($this->configCache->get('database', 'charset'));
+		$socket  = trim($this->configCache->get('database', 'socket')); 
```

Comment:
Do we need an entry at `static/defaults.config.php` as well? At least for documentation purpose